src/server/game/Tools/PlayerDump.cpp

```diff
@@ -20,378 +20,713 @@
 #include "CharacterCache.h"
 #include "Common.h"
 #include "DatabaseEnv.h"
+#include "Log.h"
 #include "ObjectMgr.h"
+#include "Player.h"
 #include "World.h"
-#include "StringConvert.h"
+#include <fstream>
+#include <sstream>
 
-constexpr auto DUMP_TABLE_COUNT = 27;
+ // static data
+enum GuidType : uint8
+{
+    // 32 bit long guids
+    GUID_TYPE_ACCOUNT,
+    GUID_TYPE_CHAR,
+    GUID_TYPE_PET,
+    GUID_TYPE_MAIL,
+    GUID_TYPE_ITEM,
+
+    // 64 bit long guids
+    GUID_TYPE_EQUIPMENT_SET,
+
+    // special types
+    GUID_TYPE_NULL // set to null
+};
```

Comment:
```suggestion
enum class GuidType : uint8
{
    // 32 bit long guids
    Account,
    Char,
    Pet,
    Mail,
    Item,

    // 64 bit long guids
    EquipmentSet,

    // special types
    Null // set to null
};
```

Reply:
if i rename it to what you suggest i will need to rename it in a few other spots as well.  Would it be necessary for me to rename them?

Reply:
I mean no, but enum class is preferred https://www.azerothcore.org/wiki/cpp-code-standards#enum-vs-enum-class

What you would need to rename them to in the rest of the code would be GuidType::Account for example

Reply:
i will do the changing up after the compile testings are done incase i need to address anything else. Thanks @Kitzunu !